src/test/java/org/apache/commons/codec/digest/MessageDigestAlgorithmsTest.java

```diff
@@ -151,6 +156,25 @@ public void testDigestFile() throws IOException {
         Assert.assertArrayEquals(digestTestData(), DigestUtils.digest(DigestUtils.getDigest(messageDigestAlgorithm),getTestFile()));
     }
 
+    @Test
+    public void testNonBlockingDigestRandomAccessFile() throws IOException {
+        Assume.assumeTrue(DigestUtils.isAvailable(messageDigestAlgorithm));
+
+        final byte[] expected = digestTestData();
+
+        Assert.assertArrayEquals(expected,
```

Comment:
In keeping with the rest of the class can you reformat the assertions to a single line. The 5 lines per assertion are very explicit.

Reply:
Fixed.